hivemq-edge/src/frontend/src/modules/Workspace/components/topics/TreeViewChart.tsx

```diff
@@ -0,0 +1,39 @@
+import { type FC } from 'react'
+import ReactTreeView, { flattenTree } from 'react-accessible-treeview'
+import { Box, HStack, Icon, Text } from '@chakra-ui/react'
+import { LuChevronDown, LuChevronRight } from 'react-icons/lu'
+
+import Topic from '@/components/MQTT/Topic.tsx'
+import { type SunburstData } from 'recharts/types/chart/SunburstChart'
+
+interface TreeViewProps {
+  data: SunburstData
+  onSelect?: (topic: string) => void
+}
+
+const TreeViewChart: FC<TreeViewProps> = ({ data }) => {
+  return (
+    <ReactTreeView
+      data={flattenTree(data)}
+      nodeRenderer={({ element, getNodeProps, level, isBranch, isExpanded }) => {
+        return (
+          <HStack {...getNodeProps()} marginLeft={8 * (level - 1)}>
```

Comment:
question(non-blocking): this `8` seems to be a magic number to add in visual difference. Maybe worth extracting to give it a name for a later view.

`const visualTabulatorSize = 8`

```ts
<HStack {...getNodeProps()} marginLeft={visualTabulatorSize * (level - 1)}>
```